packages/api/lambdas/sf-event-sqs-to-db-records/utils.js

```diff
@@ -38,24 +37,42 @@ const isPostRDSDeploymentExecution = (cumulusMessage) => {
   }
 };
 
-const hasNoParentExecutionOrExists = async (cumulusMessage, knex) => {
-  const parentArn = getMessageExecutionParentArn(cumulusMessage);
-  if (!parentArn) {
-    return true;
+const getMessageAsyncOperationCumulusId = async (cumulusMessage, knex) => {
```

Comment:
As mentioned above, I realized that I needed these helpers to return the cumulus ID, if the record exists, not just to tell me whether the record exists or not

Reply:
👍   Also if I didn't mention it before, nice job moving the selection logic out to @cumulus/message